src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -270,7 +270,7 @@ internal async Task<IEnumerable<IRpcWorkerChannel>> GetInitializedWorkerChannels
                 }
             }
             IEnumerable<IRpcWorkerChannel> workerChannels = webhostChannels == null ? _jobHostLanguageWorkerChannelManager.GetChannels() : webhostChannels.Union(_jobHostLanguageWorkerChannelManager.GetChannels());
-            IEnumerable<IRpcWorkerChannel> initializedWorkers = workerChannels.Where(ch => ch.State == RpcWorkerChannelState.Initialized);
+            IEnumerable<IRpcWorkerChannel> initializedWorkers = workerChannels.Where(ch => ch.State.HasFlag(RpcWorkerChannelState.Initialized) && ch.State.HasFlag(RpcWorkerChannelState.InvocationBuffersInitialized));
```

Comment:
>ch.State.HasFlag(RpcWorkerChannelState.Initialized) && ch.State.HasFlag(RpcWorkerChannelState.InvocationBuffersInitialized) [](start = 91, length = 123)

Once the enums are powers of two, this would become ch.State.HasFlag(RpcWorkerChannelState.Initialized | RpcWorkerChannelState.InvocationBuffersInitialized)

Reply:
got it!